### ISSUE TITLE
Screencast has undefined behaviour for non-ST screen resolutions

### ISSUE THREAD

**Issue by tin-nl** [atarijookie/ce-atari#29]
At least a warning for unsupported Screen resolutions would be nice, followed by a list of supported ones. 
Supporting higher resolutions would be nice, but ACSI bandwidth might be a very limiting factor here - the HDD should remain usable after all.

**Comment by tin-nl** [comment]
In preparation for this currently adding (aka copy&paste) SCSI support, so F030 resolution check can be added, too. Pushing the branch as soon as it's not totally broken anymore.

**Comment by atarijookie** [comment]
Great, looking forward to that :) In the meantime I'm waiting for PCBs for the Falcon SCSI adapters ;)

**Comment by tin-nl** [comment]
Woohoo! I'm desperate to get rid of that Frankencable I currently use (but don't tell Frankencable, it's pretty scary) ;-) 

This is for the external SCSI-2 conector, I presume? AFAIK @tuxie01 is working on an internal adapter atm.

**Comment by atarijookie** [comment]
> This is for the external SCSI-2 conector, I presume? 

Yes, as I assume that most of the people will be lazy to open Falcon and install the device inside. 

> AFAIK tuxie01 is working on an internal adapter 

Yeah, I saw it - he showed me ;) If there would be a demand for that one too, I could have made that one also, but now that tuxie is making them, there's no need for me to do the same thing :)

**Comment by tin-nl** [comment]
Tested on ST-ACSI, TT-ACSI, F030-SCSI. Have to test TT-SCSI.

https://github.com/atarijookie/ce-atari/tree/screencast_res
